MetaMask/metamask-extension
# Deprecating ErrorMessage in favor of BannerAlert

## Explanation

Currently, there is an outdated deprecation message assigned to ErrorMessage. This PR updates that deprecation message and updates the storybook by adding deprecated to the story title and removes the README.mdx file

* Fixes #18892

## Screenshots/Screencaps



### Before

<img width="1439" alt="Screenshot 2023-08-15 at 5 31 07 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/b8c509cc-8797-4589-a343-76ae13de9731">


### After

<img width="1440" alt="Screenshot 2023-08-15 at 5 30 06 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/42034a0e-ec6b-474c-9ae5-b9ade090a71f">

<img width="539" alt="Screenshot 2023-08-15 at 5 34 39 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/e6095b08-88d5-4157-8963-8b630e3fecd3">

## Manual Testing Steps

- Pull this branch
- Search for `<ErrorMessage` in the code base
- Hover over and see that it has been deprecated
- Run `yarn storybook`
- Search `ErrorMessage`
- See that the story has also been marked as deprecated

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
